mk/test.mk

```diff
@@ -25,12 +25,22 @@ THIS_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 _mocks: $(MOCK_FILES)
 .PHONY: _mocks
 
-test: test-hook-pre realtest
+test: check-mockgen test-hook-pre realtest
 .PHONY: test
 
 test-hook-pre:
 .PHONY: test-hook-pre
 
+check-mockgen:
+	@if [[ ! -x $(MOCKGEN) ]]; then \
+		echo ""; \
+		echo "please install mockgen using:"; \
+		echo ""; \
+		echo "    go install github.com/golang/mock/mockgen@latest"; \
```

Comment:
We should be consistent with CI Files, where we request to install 1.6.0 version of mockgen?